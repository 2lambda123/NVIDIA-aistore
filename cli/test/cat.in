echo "0123456789" > $OBJECT_1.txt
ais create bucket $BUCKET_1
ais put $OBJECT_1.txt $BUCKET_1/object_1.txt
rm -f $OBJECT_1.txt

ais cat $BUCKET_1/object_1.txt