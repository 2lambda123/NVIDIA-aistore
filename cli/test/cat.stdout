$BUCKET_1 bucket created
PUT object_1.txt into bucket $BUCKET_1
0123456789
